migration/migrating_3_4/troubleshooting.adoc

```diff
@@ -19,6 +19,7 @@ include::modules/migration-viewing-migration-crs.adoc[leveloffset=+1]
 include::modules/migration-downloading-logs.adoc[leveloffset=+1]
 include::modules/migration-restic-errors.adoc[leveloffset=+1]
 include::modules/migration-manually-rolling-back-migration.adoc[leveloffset=+1]
+include::modules/using-must-gather.adoc[leveloffset=+1]
```

Comment:
Remember to update the file name and include ref

Reply:
Fixed. Documents built successfully after renaming.